### PR TITLE
Sara / Added deriv blog single post

### PR DESCRIPTION
**Pull request by sara-fs**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/blog/template/index.js [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+import { localize, WithIntl } from 'components/localization'
+import Layout from 'components/layout/layout'
+import { SectionContainer, SEO, Show, Box, Flex } from 'components/containers'
+import { Header, Text } from 'components/elements'
+import device from 'themes/device'
+
+const HeroContainer = styled.div`
+    display: flex;
+    width: 100%;
+    max-width: 144.4rem;
```
Comment (sean-binary) [inline src/pages/blog/template/index.js:12]: We can extend `Flex` for this.

File: src/pages/blog/template/index.js [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+
+    > img {
+        max-width: 100%;
+        height: auto;
+    }
+`
+const WriterContainer = styled(Box)`
+    display: flex;
+    align-items: center;
+`
```
Comment (sean-binary) [inline src/pages/blog/template/index.js:43]: We can directly use `Flex` for this and pass in props.

File: src/pages/blog/template/index.js [hunk 3]
```diff
@@ -0,0 +1,417 @@
...
+const PublishDate = styled(Text)`
+    @media ${device.tabletL} {
+        font-size: 12px;
+    }
+`
+const MinsToRead = styled(Text)`
+    @media ${device.tabletL} {
+        font-size: 12px;
+    }
+`
```
Comment (sean-binary) [inline src/pages/blog/template/index.js:53]: These can be the same component.

File: src/pages/blog/template/index.js [hunk 4]
```diff
@@ -0,0 +1,417 @@
...
+const WriterImage = styled.div`
+    width: 4.8rem;
+    height: 4.8rem;
+    border-radius: 5rem;
+    margin-right: 8px;
+
+    @media ${device.tabletL} {
+        width: 4rem;
+        height: 4rem;
```
Comment (sean-binary) [inline src/pages/blog/template/index.js:62]: If you intended for this to be 40px. You'll need to use `px` here as 1rem does not always equal to 10px in this project.

File: src/pages/blog/template/index.js [hunk 5]
```diff
@@ -0,0 +1,417 @@
...
+            max-width: 100%;
+            height: auto;
+        }
+    }
+`
+const AuthorText = styled(Text)`
+    @media ${device.tabletL} {
+        font-size: 12px;
+    }
+`
```
Comment (sean-binary) [inline src/pages/blog/template/index.js:74]: Can be the same component as line 44.

File: src/pages/blog/template/index.js [hunk 6]
```diff
@@ -0,0 +1,417 @@
...
+const WrittenbyText = styled(Text)`
+    color: var(--color-grey-5);
+    font-size: var(--text-size-xxs);
```
Comment (sean-binary) [inline src/pages/blog/template/index.js:77]: These size variables should be deprecated, you can use `px` here or pass in as props. Can pass in `color` as props too.

File: src/pages/blog/template/index.js [hunk 7]
```diff
@@ -0,0 +1,417 @@
...
+        flex-direction: column;
+    }
+`
+const SideBarContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    max-width: 28.2rem;
+    margin-right: 12.6rem;
+    width: 100%;
+`
```
Comment (sean-binary) [inline src/pages/blog/template/index.js:96]: Can use `Flex` for this and pass props.

File: src/pages/blog/template/index.js [hunk 8]
```diff
@@ -0,0 +1,417 @@
...
+const ArticleTagContainer = styled(Box)`
+    display: flex;
+    flex-wrap: wrap;
+`
```
Comment (sean-binary) [inline src/pages/blog/template/index.js:100]: Same here.

File: src/pages/blog/template/index.js [hunk 9]
```diff
@@ -0,0 +1,417 @@
...
+const Tag = styled.div`
+    height: 2.2rem;
+    color: var(--color-blue-10);
+    font-weight: bold;
+    font-size: var(--text-size-xs);
+    border-radius: 0.8rem;
+    background-color: var(--color-blue-9);
+    padding: 1px 8px;
```
Comment (sean-binary) [inline src/pages/blog/template/index.js:108]:
```suggestion
    padding: 1px 8px 0;
```

Comment (sean-binary) [reply src/pages/blog/template/index.js:108]: We'll need this in order to center it properly.

File: src/pages/blog/template/index.js [hunk 10]
```diff
@@ -0,0 +1,417 @@
...
+    flex-direction: column;
+    max-width: 28.2rem;
+    margin-right: 12.6rem;
+    width: 100%;
+`
+const ArticleTagContainer = styled(Box)`
+    display: flex;
+    flex-wrap: wrap;
+`
+const Tag = styled.div`
```
Comment (sean-binary) [inline src/pages/blog/template/index.js:101]: Can consider using `Flex` for this.

File: src/pages/blog/template/index.js [hunk 11]
```diff
@@ -0,0 +1,417 @@
...
+        }
+    }
+    & hr {
+        margin: 32px 0;
+    }
+    & ul {
+        margin-top: 3.2rem;
+        list-style-type: disc;
+        margin-block-start: 1em;
+        margin-block-end: 1em;
```
Comment (sean-binary) [inline src/pages/blog/template/index.js:154]: Try to avoid using `em` as it isn't commonly here.

File: src/pages/blog/template/index.js [hunk 12]
```diff
@@ -0,0 +1,444 @@
...
+        height: 500px;
+        margin-bottom: 30px;
+    }
+    @media ${device.laptop} {
+        height: 450px;
+        margin-bottom: 20px;
+    }
+
+    @media ${device.tabletL} {
+        background-image: linear-gradient(#f2f3f4 80%, #ffffff 20%);
```
Comment (sean-binary) [inline src/pages/blog/template/index.js:30]:
```suggestion
        background-image: linear-gradient(var(--color-grey-8) 80%, var(--color-white) 20%);
```

File: gatsby-config.js [hunk 13]
```diff
@@ -208,5 +208,17 @@ module.exports = {
                 offset: -100,
             },
         },
+        {
+            resolve: 'gatsby-plugin-page-progress',
+            options: {
+                includePaths: ["/", { regex: "^/blog" }],
```
Comment (sean-binary) [inline gatsby-config.js:214]: `/` shouldn't be in here as this will show on the homepage as well.

File: src/pages/blog/template/index.js [hunk 14]
```diff
@@ -0,0 +1,444 @@
...
+    & h6 {
+        font-size: 16px;
+        line-height: 24px;
+        margin-top: 40px;
+        font-weight: bold;
+    }
+
+    @media (max-width: 1024px) {
+        left: 0;
+    }
```
Comment (sean-binary) [inline src/pages/blog/template/index.js:214]: Do we need this?

File: src/pages/blog/template/index.js [hunk 15]
```diff
@@ -0,0 +1,444 @@
...
+                            />
+                            <Header as="h1" type="page-title">
+                                {data?.data.article_title}
+                            </Header>
+                            <InfoText
+                                size="14px"
+                                mt="16px"
+                                dangerouslySetInnerHTML={{
+                                    __html: data?.data.minutes_to_read + ' min read',
+                                }}
```
Comment (sean-binary) [inline src/pages/blog/template/index.js:330]: We can just insert the data into the JSX normally here, like in line 323.

File: src/pages/blog/template/index.js [hunk 16]
```diff
@@ -0,0 +1,444 @@
...
+                                            />
+                                        </WriterImage>
+                                        <Box>
+                                            <WrittenbyText color="grey-5" size="12px">
+                                                {localize('Written by')}
+                                            </WrittenbyText>
+                                            <InfoText
+                                                dangerouslySetInnerHTML={{
+                                                    __html: data?.data.author,
+                                                }}
```
Comment (sean-binary) [inline src/pages/blog/template/index.js:367]: Same here.

File: src/pages/blog/template/index.js [hunk 17]
```diff
@@ -0,0 +1,444 @@
...
+                                    />
+                                </WriterImage>
+                                <Box>
+                                    <WrittenbyText color="grey-5" size="12px">
+                                        {localize('Written by')}
+                                    </WrittenbyText>
+                                    <InfoText
+                                        dangerouslySetInnerHTML={{
+                                            __html: data?.data.author,
+                                        }}
```
Comment (sean-binary) [inline src/pages/blog/template/index.js:407]: Same here.

File: src/pages/blog/template/index.js [hunk 18]
```diff
@@ -0,0 +1,428 @@
...
+    }
+    & a {
+        font-size: 20px;
+        color: var(--color-red);
+        text-decoration: none;
+        cursor: pointer;
+
+        &:hover {
+            text-decoration: underline;
+        }
```
Comment (seanho96) [inline src/pages/blog/template/index.js:153]: I think this should be nested within the `p` element.

**Pull request closed without merging**